cli/kata-env.go

```diff
@@ -27,7 +27,7 @@ import (
 //
 // XXX: Increment for every change to the output format
 // (meaning any change to the EnvInfo type).
-const formatVersion = "1.0.21"
+const formatVersion = "1.0.23"
```

Comment:
mathematics has changed since I left the school :smile:

Reply:
Haha! Well, there are two bumps to this number in this PR :)